purity_fb/README.md

```diff
@@ -1,16 +1,30 @@
-# Pure Storage FlashBlade REST 1.10 Python SDK Documentation
```

Comment:
This file looks good, but there's another copy of README.md that's in the root of this repo. That one needs to be updated as well (you can just copy this one to that location, since the contents match).

Reply:
Copied purity_fb/README.md to the level.